src/main/java/storage/Storage.java

```diff
@@ -26,9 +29,11 @@ public static void createNewFile(Path path) throws IOException {
         if (!Files.isDirectory(path.getParent())) {
             //  System.out.println("Directory not found, creating new one");
             Files.createDirectories(path.getParent());
+            logger.log(Level.INFO, "new directory created");
         }
         if (!Files.exists(path)) {
             Files.createFile(path);
+            logger.log(Level.INFO, "new tests.txt file created");
         }
     }
 
```

Comment:
Great use of logging here to note the creation of important save files and folders!